File: MobileAGECTR/ServerSide/entity/Concession.php
<?php


class Concession
{
    public $idConcession;
    public $state;
    public $customerPrice;
    public $sellingPrice;
    public $idCustomer;
    public $book;
    public $reservedExpireDate;
    public $urlPhoto;
    public $isAnnotated;
    public $expireDate;
    public $refuseReason;

    /**
     * @return mixed
     */
    public function getIdConcession()
    {
        return $this->idConcession;
    }

    /**
     * @param mixed $idConcession
     */
    public function setIdConcession($idConcession)
    {
        $this->idConcession = $idConcession;
    }

    /**
     * @return mixed
     */
    public function getState()
    {
        return $this->state;
    }

    /**
     * @param mixed $state
     */
    public function setState($state)
    {
        $this->state = $state;
    }

    /**
     * @return mixed
     */
    public function getCustomerPrice()
    {
        return $this->customerPrice;
    }

    /**
     * @param mixed $customerPrice
     */
    public function setCustomerPrice($customerPrice)
    {
        $this->customerPrice = $customerPrice;
    }

    /**
     * @return mixed
     */
    public function getSellingPrice()
    {
        return $this->sellingPrice;
    }

    /**
     * @param mixed $sellingPrice
     */
    public function setSellingPrice($sellingPrice)
    {
        $this->sellingPrice = $sellingPrice;
    }

    /**
     * @return mixed
     */
    public function getIdCustomer()
    {
        return $this->idCustomer;
    }

    /**
     * @param mixed $idCustomer
     */
    public function setIdCustomer($idCustomer)
    {
        $this->idCustomer = $idCustomer;
    }

    /**
     * @return mixed
     */
    public function getBook()
    {
        return $this->book;
    }

    /**
     * @param mixed $book
     */
    public function setBook($book)
    {
        $this->book = $book;
    }

    /**
     * @return mixed
     */
    public function getReservedExpireDate()
    {
        return $this->reservedExpireDate;
    }

    /**
     * @param mixed $reservedExpireDate
     */
    public function setReservedExpireDate($reservedExpireDate)
    {
        $this->reservedExpireDate = $reservedExpireDate;
    }

    /**
     * @return mixed
     */
    public function getUrlPhoto()
    {
        return $this->urlPhoto;
    }

    /**
     * @param mixed $urlPhoto
     */
    public function setUrlPhoto($urlPhoto)
    {
        $this->urlPhoto = $urlPhoto;
    }

    /**
     * @return mixed
     */
    public function getIsAnnotated()
    {
        return $this->isAnnotated;
    }

    /**
     * @param mixed $isAnnotated
     */
    public function setIsAnnotated($isAnnotated)
    {
        $this->isAnnotated = $isAnnotated;
    }

    /**
     * @return mixed
     */
    public function getExpireDate()
    {
        return $this->expireDate;
    }

    /**
     * @param mixed $expireDate
     */
    public function setExpireDate($expireDate)
    {
        $this->expireDate = $expireDate;
    }

    /**
     * @return mixed
     */
    public function getRefuseReason()
    {
        return $this->refuseReason;
    }

    /**
     * @param mixed $refuseReason
     */
    public function setRefuseReason($refuseReason)
    {
        $this->refuseReason = $refuseReason;
    }



}

File: MobileAGECTR/Android/app/src/main/java/com/cegeptr/projetagectr/logic/Entity/Concession.java
package com.cegeptr.projetagectr.logic.Entity;

public class Concession {

    private String state, urlPhoto, reservedExpireDate, expireDate, refuseReason;
    private double customerPrice,sellingPrice;
    private int idConcession, idCustomer;
    private Book book;
    private Boolean isAnnotated;

    public Concession(String state, String urlPhoto, String reservedExpireDate, String expireDate, String refuseReason, double customerPrice, double sellingPrice, int idConcession, int idCustomer, Book book, Boolean isAnnotated) {
        this.state = state;
        this.urlPhoto = urlPhoto;
        this.reservedExpireDate = reservedExpireDate;
        this.expireDate = expireDate;
        this.refuseReason = refuseReason;
        this.customerPrice = customerPrice;
        this.sellingPrice = sellingPrice;
        this.idConcession = idConcession;
        this.idCustomer = idCustomer;
        this.book = book;
        this.isAnnotated = isAnnotated;
    }

    public String getRefuseReason() {
        return refuseReason;
    }

    public void setRefuseReason(String refuseReason) {
        this.refuseReason = refuseReason;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getUrlPhoto() {
        return urlPhoto;
    }

    public void setUrlPhoto(String urlPhoto) {
        this.urlPhoto = urlPhoto;
    }

    public double getCustomerPrice() {
        return customerPrice;
    }

    public void setCustomerPrice(double customerPrice) {
        this.customerPrice = customerPrice;
    }

    public double getSellingPrice() {
        return sellingPrice;
    }

    public void setSellingPrice(double sellingPrice) {
        this.sellingPrice = sellingPrice;
    }

    public int getIdConcession() {
        return idConcession;
    }

    public void setIdConcession(int idConcession) {
        this.idConcession = idConcession;
    }

    public int getIdCustomer() {
        return idCustomer;
    }

    public void setIdCustomer(int idCustomer) {
        this.idCustomer = idCustomer;
    }

    public Book getBook() {
        return book;
    }

    public void setBook(Book book) {
        this.book = book;
    }

    public String getReservedExpireDate() {
        return reservedExpireDate;
    }

    public void setReservedExpireDate(String reservedExpireDate) {
        this.reservedExpireDate = reservedExpireDate;
    }

    public Boolean getAnnotated() {
        return isAnnotated;
    }

    public void setAnnotated(Boolean annotated) {
        isAnnotated = annotated;
    }

    public String getExpireDate() {
        return expireDate;
    }

    public void setExpireDate(String expireDate) {
        this.expireDate = expireDate;
    }
}
